cs/unittest/TestConfigInspector.cs

```diff
@@ -0,0 +1,121 @@
+﻿using Microsoft.VisualStudio.TestTools.UnitTesting;
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using VW;
+using VW.Labels;
+using VW.Serializer;
+
+namespace cs_unittest
+{
+    [TestClass]
+    public class TestConfigInspector
+    {
+        [TestMethod]
+        public void TestConfigParsing()
+        {
+            var str = @"
+f1
+ns1.f2(FeatureGroup = 'a')
+ns1.f3(Namespace = ""a"",Enumerize=true, FeatureGroup = 'g'  )                
+ns1.f4(Enumerize=true , FeatureGroup ='f', Order=3,AddAnchor=true)   
+ns1.ns2.f5       
```

Comment:
maybe this answers my question above. This is for ingesting the dynamic context structure?